# true 1 和 false 0 
SerialNumber: "K69668582"
TopicName: "/left_camera/image"
camera_matrix: [1748.395580, 0.000000, 597.809765,
                0.000000, 1747.891898, 509.343147,
                0.000000, 0.000000, 1.000000]
dist_coeffs: [-0.071175, 0.062294, -0.000401, -0.002200, 0.000000]
distortion_model: "plumb_bob"

width: 1224             #图像宽度
height: 1024            #图像高度
Offset_x: 0             #X方向偏置
Offset_y: 0             #Y方向偏置

FrameRateEnable: true   #使能 
FrameRate: 10           #帧率

ExposureTime: 5000     #曝光时间

GammaEnable: true
Gamma: 0.7              #伽马因子

GainAuto: 2             #亮度的固定增益

SaturationEnable: true
Saturation: 128          #饱和度
